examples/i18n/src/lib.rs

```diff
@@ -0,0 +1,123 @@
+use seed::{prelude::*, *};
+
+use fluent::fluent_args;
+use strum::IntoEnumIterator;
+
+mod i18n;
+use crate::i18n::{translate, I18n, Lang};
+use std::str::FromStr;
+
+mod resource;
```

Comment:
I suggest to regroup it a bit, something like:
```rust
use seed::{prelude::*, *};
use std::str::FromStr;

use fluent::fluent_args;
use strum::IntoEnumIterator;

mod i18n;
use i18n::{translate, I18n, Lang};   // <- you don't have to write "crate::" prefix
mod resource;
```

Reply:
To be consequent this means "use seed ..." is now in the second block. Done.